diff --git a/CMakeLists.txt b/CMakeLists.txt
index dfad39b72..e19171886 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,6 +62,8 @@ include(oe_osx)
 include(oe_unix)
 include(oe_win32)
 
+set(INSTALL_PLUGINS_FOLDER "plugins")
+set(INSTALL_LIBRARY_FOLDER "lib")
 # Build options ..........................................................
 
 option(OSGEARTH_BUILD_TOOLS "Build the osgEarth command-line tools" ON)
